pkg/detectors/falsepositives.go

```diff
@@ -151,22 +152,30 @@ func FilterResultsWithEntropy(ctx context.Context, results []Result, entropy flo
 }
 
 // FilterKnownFalsePositives filters out known false positives from the results.
-func FilterKnownFalsePositives(ctx context.Context, detector Detector, results []Result, shouldLog bool) []Result {
+func FilterKnownFalsePositives(ctx context.Context, detector Detector, results []Result, retainFalsePositives bool) []Result {
+	if retainFalsePositives {
+		return results
+	}
```

Comment:
But why not just not call the function at all? (I.e., put this logic at call sites?) Then you can eliminate a parameter.